cime_config/usermods_dirs/newton_krylov_spinup/user_nl_clm

```diff
@@ -0,0 +1,15 @@
+hist_dov2xy = .true.,.false.
+hist_nhtfrq = 0,-175200
```

Comment:
@wwieder the above two lines set the auxiliary history file to 20-yr average vector output.

`nhtfrq` is in hours: 24 hrs/day * 365 days/yr * 20 yrs = 175200

Two lines down is the list of variables, and I think I got all of them.